core/src/dpdk/mod.rs

```diff
@@ -33,6 +33,132 @@ impl rte_ipv4_hdr {
     }
 }
 
+#[cfg(not(dpdk_ge_2411))]
```

Comment:
@tbarbette @thegwan these refactors to account for the interface changes LGTM, but checking if either of you have any concerns from working on the original DPDK layer?